### ISSUE TITLE
Handle empty state in the editor

### ISSUE THREAD

**Issue by johno** [blocks/blocks#59]

**Comment by johnletey** [comment]
Another "empty state" to handle, is when selecting the `Blocks.Root` element.

![Kapture 2019-11-26 at 8 25 29](https://user-images.githubusercontent.com/30328854/69611756-6656fd00-1026-11ea-97f0-207d61059b4d.gif)

**Comment by zaguiini** [comment]
I think this is the same issue, but the demo crashes if I remove the `HeaderBasic` component (making the desktop empty) with the following stacktrace:

```ReferenceError: _extends is not defined
    at eval (eval at <anonymous> (inline-render.js:12), <anonymous>:8:30)
    at react-beautiful-dnd.esm.js:8467
    at mi (react-dom.production.min.js:3658)
    at Rl (react-dom.production.min.js:6660)
    at Au (react-dom.production.min.js:5696)
    at Pu (react-dom.production.min.js:5685)
    at wu (react-dom.production.min.js:5441)
    at react-dom.production.min.js:2877
    at KqkS.t.unstable_runWithPriority (scheduler.production.min.js:274)
    at $a (react-dom.production.min.js:2842)
```

That I don't think helps much :D

**Comment by johnletey** [comment]
Another day, another empty state that we need to deal with 😁

![Kapture 2019-11-27 at 15 04 09](https://user-images.githubusercontent.com/30328854/69734691-56731200-1127-11ea-8900-de0161d37e29.gif)

**Comment by mohinderps** [comment]
In initial-render.js

code refers to the following:

"const BLOCKS_Container = () => jsx(BLOCKS_Root, null);

const BLOCKS_Root = ({
  children
}) => {
  return jsx(BLOCKS_Droppable, {
    droppableId: "root"
  }, (provided, snapshot) => jsx("div", **_extends(**{}, provided.droppableProps, {
    style: {
      minHeight: 'calc(100vh - 43px)'
    },
    ref: provided.innerRef
  }), children, provided.placeholder));
};"

Which tries to access _extends, which is clearly not defined in this case. So the code breaks.

**Comment by johno** [comment]
Yeah, I'm really perplexed that the `_extends` polyfill isn't being added 🤔. I wonder if it has something to do with the usage of `@babel/template` when injecting `BLOCKS_Root`?

https://github.com/blocks/blocks/blob/94f881cfb77f9bb02e9af8aae4b1105de4295a8b/packages/blocks-ui/src/babel-plugins/inject-blocks-root.js#L3-L23

**Comment by johno** [comment]
After a bit of digging it's definitely a result of the above `template.ast` call. For whatever reason, `@babel/template` transforms the spread into the polyfilled version but doesn't add the polyfill itself. 🤔
